ur_ros2_control_demos/urdf/ur5.urdf

```diff
@@ -0,0 +1,241 @@
+<?xml version="1.0" ?>
```

Comment:
Please create an issue to delete this duplicate urdf (or just delete it now). It's a duplicate of the one in ur_description/urdf

Reply:
It is not the same, it has included ros2_control macro. Will hold it for know